lib/sass/scss/parser.rb

```diff
@@ -589,6 +593,14 @@ def supports_operator
         cond
       end
 
+      def supports_declaration
+          return if tok?(/\(/)
+          name = sass_script(:parse)
+          tok!(/:/); ss
+          value = sass_script(:parse)
+          Sass::Supports::Declaration.new(name, value)
```

Comment:
Now that this is factored out, can you re-use it in `supports_condition_in_parens`?